test/jdk/java/util/Properties/PropertiesStoreTest.java

```diff
@@ -46,12 +47,18 @@
 /*
  * @test
  * @summary tests the order in which the Properties.store() method writes out the properties
- * @bug 8231640
- * @run testng PropertiesStoreTest
+ * @bug 8231640 8282023
+ * @modules jdk.localedata
```

Comment:
OK, you can remove `@modules jdk.localedata` so that other tests would run. (it'd be unusual setup, but still possible with jlink)

Reply:
Thank you Naoto. I've removed this in the latest version of the PR.